ol7/profiles/sap.profile

```diff
@@ -15,3 +15,5 @@ selections:
     - no_rsh_trust_files
     - package_ypbind_removed
     - package_ypserv_removed
+    - var_accounts_authorized_local_users=ol7forsap
```

Comment:
As the `var_accounts_authorized_local_users` contains a regular expression, could you please reflect this in the variable name by adding the `_regex` suffix? We already follow that pattern e.g. in `shared/checks/oval/rsyslog_files_ownership.xml`.